src/contract-ui/tabs/edit-extensions/components/InstalledExtensionsTable.tsx

```diff
@@ -0,0 +1,347 @@
+        title: "Extension Removed successfully",
+        status: "success",
+        duration: 5000,
+        isClosable: true,
+      });
+
+      props.onRemoveExtension();
+    } catch (error) {
+      toast({
+        variant: "solid",
+        position: "bottom",
+        title: "Failed to remove extension",
+        status: "error",
+        duration: 5000,
+        isClosable: true,
+      });
+
+      console.error("Error during uninstallation:", error);
+    }
+  };
+
+  if (!contractInfo) {
+    return <SkeletonRow ownerAccount={ownerAccount} />;
+  }
+
+  return (
+    <Tr borderBottomWidth={1} _last={{ borderBottomWidth: 0 }}>
+      <TableData>
+        <Text color="heading">{contractInfo.name}</Text>
+      </TableData>
+      <TableData>
+        <Text>{contractInfo.description || "..."}</Text>
+      </TableData>
+      <TableData>
+        <AddressCopyButton size="xs" address={contractInfo.publisher} />
+      </TableData>
+      <TableData>
+        <AddressCopyButton size="xs" address={extensionAddress} />
+      </TableData>
+
+      {/* Version */}
+      <TableData>
+        <Text>{contractInfo.version}</Text>
+      </TableData>
+
+      {/* Remove */}
+      {ownerAccount && (
+        <TableData>
+          <Flex>
+            <Tooltip
+              label="Remove"
+              bg="backgroundHighlight"
+              border="1px solid"
+              borderColor="borderColor"
+              color="accent.500"
+              rounded="xl"
+              placement="top"
+              fontSize="small"
+              px={3}
+              py={2}
+            >
+              <Button
+                isLoading={uninstallMutation.isLoading}
+                onClick={onOpen}
+                color="red.500"
+                variant="outline"
+                _hover={{
+                  bg: "accent.100",
+                }}
+                rounded={"xl"}
+                p={2}
+              >
+                <FaRegTrashAlt
+                  style={{
+                    width: "16px",
+                    height: "16px",
+                  }}
+                />
+              </Button>
+            </Tooltip>
+          </Flex>
+        </TableData>
+      )}
+
+      <Modal isOpen={isOpen} onClose={onClose} isCentered>
+        <ModalOverlay />
+        <ModalContent>
+          <form
+            onSubmit={(e) => {
+              e.preventDefault();
+              handleRemove();
+            }}
+          >
+            <ModalHeader fontSize={"larger"} pb={1}>
+              Uninstall Extension
+            </ModalHeader>
+            <ModalCloseButton />
+            <ModalBody>
+              Are you sure you want to uninstall{" "}
+              <Box as="span" fontWeight={500} color="heading">
+                {contractInfo.name}
+              </Box>{" "}
+              ?
+            </ModalBody>
+
+            <ModalFooter pt={10} gap={3}>
+              <Button
+                type="submit"
```

Comment:
this should *not* be type submit, since that would submit the form

Reply:
good catch, setting type="button"